x11-misc/compiz-debug-utils/metadata.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>elliot@futuramerlin.com</email>
```

Comment:
a developer and/or proxy-maint@gentoo.org is missing. Please discuss with the proxy team.